proto/v1/actuator_service.proto

```diff
@@ -141,4 +141,7 @@ message ActuatorInfo {
   bool iam_guard = 18;
 
   repeated string unlicensed_features = 19;
+
+  // logo is the branding logo.
+  string logo = 20;
```

Comment:
Two thoughts
1) I wonder if it's quite expensive to include a picture in the actuator.get call that's quite frequent and common;
2) Should this be "bytes" instead of string?

Reply:
I'm also considering adding a new API in the setting service like "GetBranding". WDYT

Reply:
SG, or call it GetResourcePackage(). In case we need to support theme like resources.

Reply:
Put the new API in the actuator service instead.